embedding.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
+from dotenv import load_dotenv
+import os
+
+
+class EmbeddingType(Enum):
+    OPENAI = 'openai'
+    HUGGINGFACE = 'hugging_face'
```

Comment:
I think embedding name for KoSimCSE Embedding must be kosimcse or etc.
'HUGGINGFACE' is so confusing becuase there are some basic embedding models from huggingface itself.